release/models/aft/openconfig-aft-common.yang

```diff
@@ -238,6 +244,19 @@ submodule openconfig-aft-common {
           }
         }
 
+        container mpls-in-udp {
+          description
+            "When specified, the packet has an MPLS in UDP header applied to it before
+            forwarding to the specified next-hop.";
```

Comment:
I think here we are implying that `encapsulate-header` should be set to `MPLS_IN_UDPV[46]` before this container is used, right?

Reply:
(This would be consistent with the `encapsulate-header` being set to `IPV4` when we do IPinIPv4.)

Reply:
> I think here we are implying that `encapsulate-header` should be set to `MPLS_IN_UDPV[46]` before this container is used, right?

Yes, like this snippet:
```yaml
openconfig-network-instance:
  network-instances: 
    - network-instance: "customerX_vpc1"
      afts: 
        next-hops:
          - next-hop: 123
            state:
              encapsulate-header: openconfig-aft-types:MPLS_IN_UDPV6
```

Reply:
I'll update the description to reflect that this is a container and it is used when encapsulate-header is set to openconfig-aft-types:MPLS_IN_UDPV(4|6)